Does not detect non-standard private lists

When committing https://github.com/apache/tooling-agenda/commit/34fa814fc94ce464e85407a419b15a4417a2acee

I got this email:

```
An error occurred while processing the notifications feature in .asf.yaml: 

[ERROR] The mailing list target for notifications::commits MUST be a private mailing list.

---
With regards, ASF Infra.
```

--- a/asfyaml/feature/notifications.py
+++ b/asfyaml/feature/notifications.py
@@ -59,6 +59,7 @@ VALID_PRIVATE_TARGETS = [
     "private@*",
     "security@*",
     "commits@infra.apache.org",
+    "private-commits@tooling.apache.org",
 ]
 
 # regex for valid ASF mailing list

--- a/asfyaml/feature/notifications.py
+++ b/asfyaml/feature/notifications.py
@@ -59,7 +59,7 @@ VALID_PRIVATE_TARGETS = [
     "private@*",
     "security@*",
     "commits@infra.apache.org",
-    "private-commits@tooling.apache.org",
+    "private-commits@*",
 ]
 
 # regex for valid ASF mailing list

